### PR TITLE
Add Shiba app

### PR DESCRIPTION
**Pull request by xHN35RQ**
This pull request is better than my last attempt. Thanks for fixing that one for me.

Shiba is a really sweet Markdown preview app.

https://github.com/rhysd/Shiba

### CODE REVIEW

**Comment by xHN35RQ** [comment]
wait let me squash a commit DAMN WHITESPACE

File: readme.md [hunk 1]
```diff
@@ -48,6 +48,7 @@ Some good apps written with Electron.
 - [Yeoman](https://github.com/yeoman/yeoman-app) - Scaffold projects using Yeoman.
 - [Mojibar](https://github.com/muan/mojibar) - Emoji searcher in the menubar.
 - [Markn](https://github.com/minodisk/markn) - Lightweight Markdown viewer.
+- [Shiba](https://github.com/rhysd/Shiba) - Markdown live preview app with linter.
```
Comment (sindresorhus) [inline readme.md]: > Live preview markdown app with linting

**Comment by xHN35RQ** [comment]
Done. Thanks.

**Comment by xHN35RQ** [comment]
:1:

**Pull request merged**